src/libopensc/pkcs15-piv.c

```diff
@@ -359,7 +359,7 @@ static int sc_pkcs15emu_piv_init(sc_pkcs15_card_t *p15card)
 	};
 
 	static const pindata pins[] = {
-		{ "01", "PIV Card Holder pin", "", 0x80,
+		{ "01", "App PIN", "", 0x80,
```

Comment:
This part of prompt is shortened to fit the cardholder name into it. It is passed through the Label of the `CK_TOKEN_INFO` structure, which is limited to 32 characters.
http://docs.oasis-open.org/pkcs11/pkcs11-curr/v2.40/cs01/pkcs11-curr-v2.40-cs01.html